README.md

```diff
@@ -256,8 +256,8 @@ It was called 'shepherd' at first because it was helpful for dealing with *lamb*
 
 Pull requests welcome!
 
-Test: `npm test`
+Test: `yarn test`
 
-Rebuild on file change: `npm run compile -- -w`
+Rebuild on file change: `yarn compile -- -w`
 
-Publish: `npm run pub` "publish" is reserved by npm
+Publish: `yarn pub` "publish" is reserved by npm
```

Comment:
yes. You can't overwrite it. fwiw `yarn publish` is better than `npm publish`. It has a np-lite kind of workflow